iohk/NixOps.hs

```diff
@@ -73,6 +75,9 @@ projectURL     IOHK            = "https://github.com/input-output-hk/iohk-nixops
 projectURL     Nixpkgs         = "https://github.com/nixos/nixpkgs.git"
 projectURL     Stack2nix       = "https://github.com/input-output-hk/stack2nix.git"
 
+nixpkgsRevTarball :: Commit -> URL
+nixpkgsRevTarball (Commit rev) = URL $ "https://github.com/NixOS/nixpkgs/archive/"<>rev<>".tar.gz"
```

Comment:
1. the `set-rev` command currently accepts a revision argument
2. it takes the GH owner and repo implied from the default URLs for the projects it's set-revving -- which are basically strings now
3. if we choose to parametrise owner and repo, we either require them (bad) or provide defaults (good)
4. the good option is more involved